tdm-fe/src/app/components/layout/WebServicesHeader.tsx

```diff
@@ -0,0 +1,21 @@
+import '~/app/components/layout/scss/WebServicesHeader.scss';
+import Container from '@mui/material/Container';
+
+const WebServicesHeader = () => {
+    // Todo add the correcte text when available
+    return (
+        <div id="ws-header">
+            <Container>
+                <p className="text">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sit amet vulputate ex. Vivamus vitae
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** ‏There should be a real sentence. :P

Reply:
Communication departments are working on this.